python_modules/libraries/dagster-airlift/dagster_airlift/core/load_defs.py

```diff
@@ -344,10 +350,13 @@ def replace_assets_in_defs(
 def enrich_airflow_mapped_assets(
     mapped_assets: Sequence[MappedAsset],
     airflow_instance: AirflowInstance,
+    source_code_retrieval_enabled: Optional[bool] = None,
```

Comment:
seems like we have the `= None` default in a lot of places, are all of these user-facing APIs? would be safer to not pass a default and explicitly thread through the value if possible, seems very easy to miss passing this at an internal callsite if it has a default

Reply:
yea good point.

Reply:
(most are not user facing APIs)